ci-operator/step-registry/ipi/install/hosted-loki/ipi-install-hosted-loki-ref.yaml

```diff
@@ -0,0 +1,14 @@
+ref:
+  as: ipi-install-hosted-loki
+  from: cli
+  commands: ipi-install-hosted-loki-commands.sh
+  resources:
+    requests:
+      cpu: 100m
+      memory: 100Mi
+  credentials:
+  - namespace: test-credentials
+    name: loki-stage-collector-test-secret
+    mount_path: /var/run/loki-secret
+  documentation: |-
+    The IPI installs promtail confogured to send data to Loki hosted on Observatorium.
```

Comment:
nit: "confogured" -> "configured".  Is there anything IPI-specific about this config?  If it's just dropping day-1 manifests in place for the installer to slurp up, I expect we should have something like:

> Adds manifests that will configure promtail to send data to Loki hosted on Observatorium.

or some such, and similarly for the existing `ci-operator/step-registry/ipi/install/loki/ipi-install-loki-ref.yaml`.

Reply:
> Is there anything IPI-specific about this config? 

No, not really, this may work on UPI as well. Most likely proxy jobs would require additional images cloned (we pull CLI and promtail from docker.io), but that's something for the future.

Keeping this in `ipi` for now, so that we could slowly rollout *pi-less version of this step later.